public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
+    event.dataset.height = event.clientHeight
+    event.style.height = 0
+})
+
+document.querySelectorAll(".event").forEach(function(event) {
+    let clicked = 0
+    event.addEventListener('click', function() {
+        clicked++
+        let lower = this.querySelector('.lower')
+        let arrow = this.querySelector(".arrowBox")
+        if(clicked % 2) {
+            arrow.innerHTML = "<span>&#x21E7;</span>"
+
+            var id = setInterval(function() {
+                if (lower.clientHeight < lower.dataset.height) {
+                    lower.style.height = (parseInt(lower.style.height) + 15) + 'px'
+                } else {
+                    clearInterval(id)
+                }
+            }, 16)
+        }
+        else {
+            arrow.innerHTML = "<span>&#x21E9;</span>"
+
+            var id = setInterval(function() {
+                if (lower.clientHeight > 0) {
+                    lower.style.height = (parseInt(lower.style.height) - 10) + 'px'
+                } else {
+                    clearInterval(id)
+                }
```

Comment:
These intervals cause very funky things to happen when the height of the element isn't divisible by 10 and/or there's lots of events on the screen at the same time.. Either fix this or use CSS transitions by applying a class with height: 0 when the info is hidden and height: auto when the info is visible.